static/css/locus68.css

```diff
@@ -69,6 +69,9 @@ h1, h2 {
 .img-circle {
   border-radius: 50%;
 }
+.img-circle:hover {
```

Comment:
`img-circle` is used for other places than the avatars in the settings menu. We should instead modify the `.select-avatar` class: https://github.com/TheOrangeOne/locus68/blob/0067061defe13bfe69cfa4dae5d40dc8a3a3177b/static/css/locus68.css#L226-L229

good idea though!